mesh_file_export.py

```diff
@@ -52,18 +54,45 @@ def clean_name(name):
 
 
 def get_export_armature(collection):
-    
-    if(collection.hide_viewport):
+
+    if (collection.hide_viewport):
         return None
 
     for object in collection.objects:
-        if(object.type == 'ARMATURE' and object.hide_viewport == False):
+        if (object.type == 'ARMATURE' and object.hide_viewport == False):
             return object
-            
+
     for child in collection.children:
         return get_export_armature(child)
 
 
+def split_UV_borders():

```

Comment:
There might be implementations of this in some of the edits people have made to gaukler's alo exporter, though they are probably buried in the eaw modding discord. Also relying on a specific view to exist is a bit painful. But if it works, it works.